samples/nrf9160/lwm2m_client/src/sensors/light_sensor.c

```diff
@@ -38,7 +38,7 @@ static const struct device *light_sensor_dev = DEVICE_DT_GET(DT_NODELABEL(bh1749
 static int64_t fetch_timestamp;
 
 
-static int sensor_read(struct sensor_value measurements[])
+static int _sensor_read(struct sensor_value measurements[])
```

Comment:
Maybe this should be given a more specific name? Like `light_sensor_read` or something?

Reply:
The thing is, this function is not specific. There already is `light_sensor_read()` in this file, along with `color_sensor_read()`, and this helper func is called from both. I'm open for other name suggestions though.

Reply:
This is a common and quick way to fix a name conflict and it is used elsewhere too. I think that it is fine for an upmerge.

Reply:
Oh my bad, I hadn't noticed those other naming conflicts. I don't have a better alternative then, I'll resolve